src/opencl_bitcoin_fmt_plug.c

```diff
@@ -459,17 +443,19 @@ static int cmp_one(void *binary, int index)
 
 static int cmp_exact(char *source, int index)
 {
-	return cracked[index];
+	return 1;
 }
 
 static void bitcoin_set_key(char *key, int index)
```

Comment:
BTW, no reason for `bitcoin_` in the `bitcoin_set_key` name if we don't share this function with the CPU format.